v1/pkg/client/api/openapi.yaml

```
@@ -2893,6 +2893,9 @@ components:
         Description: Description of new Host
         NetworkUntagged: 046b6c7f-0b8a-43b9-b35d-6489e6daee92
         UserData: UserData
+        WWPNs:
+        - WWPNs
+        - WWPNs
```

This doesn't look right. Each entry is one WWPN, not WWPNs. Not sure how this got generated this way. Perhaps add real WWPN examples in the yaml files.
```suggestion
        WWPNs:
        - WWPN
        - WWPN
```

---

Hi @mchuang3, In the model/host.go, it is defined as array of string **WWPNs []string**. So I added it here in the same way. And I assume that WWPNs are the read only member and not with writeable.

---

Yes, it's read-only. However, "WWPN" or "WWPNs" are not good examples. Please use a real example of WWPN, similar to UUID examples in the other Host attributes.